vip_spec.xsd

```diff
@@ -113,12 +113,10 @@ reference elements for polling location
         <xs:element name="ElectionAdministration">
           <xs:complexType>
             <xs:all>
-              <xs:element name="Name" type="InternationalizedText" minOccurs="0" />
-              <xs:element name="ElectionOfficialPersonId" type="xs:IDREF" minOccurs="0" />
-              <!-- Q: Please remind me what OVC stands for so we can expand this. -->
+              <!-- Q: Please remind me what OVC stands for so we can expand this,
+                      and should we move it to Department?
+              -->
```

Comment:
Regarding the second part of the question, you may not need to if `ElectionAdministration.Department` is unbounded. A single `ContactId` per `ElectionAdministration.Department` may serve that purpose if the departments are individually defined.